### ISSUE TITLE
`skip` option is missing

### ISSUE THREAD

**Issue by yegor256** [volodya-lombrozo/jtcop#414]
I'm trying to do this:

```
          <plugin>
            <groupId>com.github.volodya-lombrozo</groupId>
            <artifactId>jtcop-maven-plugin</artifactId>
            <version>1.2.4</version>
            <executions>
              <execution>
                <phase>verify</phase>
                <goals>
                  <goal>check</goal>
                </goals>
                <configuration>
                  <skip>true</skip>
                </configuration>
              </execution>
            </executions>
          </plugin>
```

However, it doesn't work. The `skip` parameter is a good practice for Maven plugins.

**Comment by yegor256** [comment]
@volodya-lombrozo please, pay attention to this

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you try it one more time with `1.3.1` version please? I added `skip` option.

**Comment by yegor256** [comment]
@volodya-lombrozo still not there: https://repo1.maven.org/maven2/com/github/volodya-lombrozo/jtcop-maven-plugin/

**Comment by yegor256** [comment]
@volodya-lombrozo still nothing. are you sure it's released?

**Comment by volodya-lombrozo** [comment]
@yegor256 Thank you! Actually I had some problems with authetification in Nexus. So, I fixed the problem and released the `1.3.1` version properly and It has already appear at https://repo1.maven.org/maven2/com/github/volodya-lombrozo/jtcop-maven-plugin/. Could you try one more time, please?

**Comment by yegor256** [comment]
@volodya-lombrozo now it works, thanks!
